generators/entity/index.js

```diff
@@ -238,6 +238,38 @@ module.exports = EntityGenerator.extend({
             this.skipServer = true;
         }
     },
+
+    _loadOtherEntityJson: function(entityName) {
```

Comment:
yes, i just realized that. i just removed that and used getEntityProperty from the base generator.